conftest.py

```diff
@@ -17,6 +17,18 @@
     raise('Your version of pytest is too old, you should have at least '
           'pytest >= {} installed.'.format(PYTEST_MIN_VERSION))
 
+
+def pytest_addoption(parser):
+    parser.addoption("--skip-network", action="store_true",
+                     help="skip network tests")
+
+
+def pytest_runtest_setup(item):
+    if 'network' in item.keywords and item.config.getoption("--skip-network"):
```

Comment:
Does this run for you locally? Travis CI doesn't seem to be happy although it is the same setup as pandas..

Reply:
Yep it passes locally. I did not find to reproduce the error yet. I should try the to reproduce the testing environment.

Reply:
I follow the pytest documentation. It seems that this is working but I did not find the root of the previous issue.